From: Jonathan Corbet <corbet@lwn.net>
Date: Thu, 24 Aug 2017 16:11:09 -0600
Subject: [PATCH] scsi: Fix the kerneldoc for scsi_initialize_rq()
References: bsc#1077989
Git-commit: 35c0506f27f6e3f278592d631901163cbccce28d
Patch-mainline: v4.14-rc1

The kerneldoc comment for scsi_initialize_rq() neglected to document the
"rq" parameter, leading to this docs build warning:

  ./drivers/scsi/scsi_lib.c:1116: warning: No description found for parameter 'rq'

Document the parameter and make the build slightly quieter.

[mkp: used wording suggested by Bart]

Signed-off-by: Jonathan Corbet <corbet@lwn.net>
Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 drivers/scsi/scsi_lib.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/scsi/scsi_lib.c b/drivers/scsi/scsi_lib.c
index 22c12d560ea7..ad2c2c5a990d 100644
--- a/drivers/scsi/scsi_lib.c
+++ b/drivers/scsi/scsi_lib.c
@@ -1114,6 +1114,7 @@ EXPORT_SYMBOL(scsi_init_io);
 
 /**
  * scsi_initialize_rq - initialize struct scsi_cmnd.req
+ * @rq: Request associated with the SCSI command to be initialized.
  *
  * Called from inside blk_get_request().
  */
-- 
2.12.3

